releasenotes/releasenotes_V0.16.0.md

```diff
@@ -0,0 +1,16 @@
+# Release Notes 0.16.0
+
+## New Features
+
+- No new features
+
+## Fixed Issues
+
+- No fixed issues
+
+## Other Changes
+
+The dynatrace-service has absorbed the functionality of the dynatrace-sli-service. 
```

Comment:
Ticket number was https://github.com/keptn-contrib/dynatrace-service/issues/363 if you like to add this info.

Reply:
Yes, I will add this as it's an important detail.

Reply:
Updated with the latest push